docs/modules/ROOT/pages/lua/examples/paint.adoc

```diff
@@ -84,6 +85,26 @@ assert(screen,'No <FINComputerScreen> PCI device found')
 
 [source,Lua]
 ----
+local gpu = computer.getPCIDevices(classes.GPU_T1_C)[1]
+
+assert(gpu,'No <GPU_T1_C> PCI device found')
+
+
+local screen = computer.getPCIDevices(classes.FINComputerScreen)[1]
+
+local comp_screen = component.findComponent(classes.Screen)[1]
+
+if comp_screen then
+
+    screen = component.proxy(comp_screen)
+
+    assert(screen,'Failed to proxy <Screen> component')
+
+end
+
+assert(screen,'No <Screen> nor <FinComputerScreen> component found')
+
+
 -- [ GPU Setup ]
```

Comment:
Once again... why all that empty lines? Thats soo much wasted space.

Reply:
Readability .. ? 

Those couple `\n` bytes won't hurt anyone.

Reply:
IMO all those empty lines make it a lot less readable. It's hard to see what is grouped together.
My ideal code of that would be:
```lua
local gpu = computer.getPCIDevices(classes.GPU_T1_C)[1]
assert(gpu,'No <GPU_T1_C> PCI device found')

local screen = computer.getPCIDevices(classes.FINComputerScreen)[1]
local comp_screen = component.findComponent(classes.Screen)[1]

if comp_screen then
    screen = component.proxy(comp_screen)
    assert(screen,'Failed to proxy <Screen> component')
end

assert(screen,'No <Screen> nor <FinComputerScreen> component found')
```
you can see what lines of code group together to provide a given feature... the first group queries and checks the gpu
the second group queries the possible screens, the third group decides on what screen to use and the last group does the final validation.

Reply:
> Readability .. ?
> 
> Those couple `\n` bytes won't hurt anyone.

I saw this comment earlier today but just now got home from work and I must agree with panakotta. Its much harder to read with the empty spaces. Either remove them or fill them with comments to help educate.